examples/jenkins/README.md

```diff
@@ -53,23 +53,29 @@ Steps
 
     **Note**: This template uses an EmptyDir type volume.  If you want to ensure your jenkins configuration/job information is persisted through pod restarts and deployments, you can use the jenkins-persistent-template.json template file which uses a persistent volume but requires additional [PersistentVolume](https://docs.openshift.org/latest/admin_guide/persistent_storage_nfs.html) setup.  
     
+    **Note**: Using the default Jenkins template, with the new OpenShift Login plugin, you should be able to provide any non-empty password for the `admin` user since the default OpenShift identity provider is `Allow All`.
```

Comment:
now i'm especially confused by this.  does this mean that because the jenkins image defines an "admin" user, then when you use the openshift oauth plugin, you can login as the "admin" user but w/o specifying the proper admin password, and you end up logged into jenkins as "admin" with admin authority?

I think i need an explanation of how "jenkins" users intersect with "openshift" users, but this sounds bad.  I would expect the admin user to have no privileges because the "openshift" admin user has no privileges in the namespace (doesn't even exist).